From: Jiri Slaby <jslaby@suse.cz>
Date: Wed, 13 Jul 2011 21:33:30 +0200
Subject: kabi: fix jiffies_to_clock_t breakage
Patch-mainline: never

Upstream commit ec9ee3fc9e54 (time: Change jiffies_to_clock_t()
argument type to unsigned long) changed jiffies_to_clock_t parameter
from long to ulong. It means nothing to the callsites. The register
is of the same length.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/linux/jiffies.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/include/linux/jiffies.h
+++ b/include/linux/jiffies.h
@@ -303,7 +303,11 @@ extern void jiffies_to_timespec(const un
 extern unsigned long timeval_to_jiffies(const struct timeval *value);
 extern void jiffies_to_timeval(const unsigned long jiffies,
 			       struct timeval *value);
+#ifdef __GENKSYMS__
+extern clock_t jiffies_to_clock_t(long x);
+#else
 extern clock_t jiffies_to_clock_t(unsigned long x);
+#endif
 extern unsigned long clock_t_to_jiffies(unsigned long x);
 extern u64 jiffies_64_to_clock_t(u64 x);
 extern u64 nsec_to_clock_t(u64 x);
